src/deluge/gui/views/view.cpp

```diff
@@ -1350,8 +1350,8 @@ void View::drawOutputNameFromDetails(int outputType, int channel, int channelSuf
 		else {
 			OLED::drawString(nameToDraw, 0, yPos, OLED::oledMainImage[0], OLED_MAIN_WIDTH_PIXELS, textSpacingX,
 			                 textSpacingY);
-			OLED::setupSideScroller(0, name, 0, OLED_MAIN_WIDTH_PIXELS, yPos, yPos + textSpacingY, textSpacingX,
-			                        textSpacingY, false);
+			//OLED::setupSideScroller(0, name, 0, OLED_MAIN_WIDTH_PIXELS, yPos, yPos + textSpacingY, textSpacingX,
```

Comment:
I think it would be a more useful option for the menu if it still were to scroll the full name _once_ and then stop. Although this doesn't seem to be an option in the function signature yet..

Reply:
Agreed. I'm am a relative novice and implemented this change after seeing what minor things I could understand enough to modify without breaking anything. Any suggestions or modifications are welcome, and scrolling once would be better than no scrolling (though I prefer this to infinite scrolling personally)